jimp.d.ts

```diff
@@ -519,8 +519,8 @@ declare namespace Jimp {
             g: number,
             b: number,
             a: number,
-            cb: GenericCallback<number, T, Jimp>
-        ): this;
+            cb: GenericCallback<number, any, Jimp>
+        ): number;

```

Comment:
This should actually return `void`

Reply:
this one for sure returns a number though

Reply:
Well [the function](https://github.com/oliver-moran/jimp/blob/master/src/index.js#L870) doesn't have a return statement

Reply:
I think you're reading the diff wrong. line 523 is a part of `rgbaToInt` not `appendConstructorOption`

Reply:
Oh 100% you're right, sorry about that 🤦‍♂️